docs/index.rst

```diff
@@ -69,6 +69,35 @@ Example:
    to this mount.
 
 
+CDN
```

Comment:
could you please update by saying that token auth part is purely for Akamai CDN.

Reply:
..and looks like just for docker v2 content

Reply:
I clarified that in the docs now. Thanks!